src/js/python/assets.js

```diff
@@ -61,7 +61,7 @@ export default () => {
         },
         {
             "content": readAssetFile("python/ml", "pyml:post_processing:parity_plot:matplotlib.j2.py"),
-            "name": "post_processing_parity_plot_matplotlib.py",
+            "name": "post_processing:parity_plot_matplotlib.py",
```

Comment:
`post_processing:parity_plot:matplotlib`

Reply:
Fixed in recent commit